benchmark/utils.h

```diff
@@ -32,14 +32,20 @@ std::string toString(const TransposeParams& params);
 std::string toString(const std::shared_ptr<HeuristicParams>& params);
 std::string toString(LaunchParams lparams);
 
-// Run benchmark iterations with provided inputs. If not segmented, report
-// kernel time from the runtime, as well as heuristic parameters. If segmented
-// use timers. Make sure to clear L2 between iterations.
+// Run benchmark iterations with provided inputs. Report kernel time
+// from the runtime, as well as heuristic parameters if not segmented.
```

Comment:
Add more: if segmented will report the total kernel time of all the segmented kernels.

Reply:
Updated